dbt/compilation.py

```diff
@@ -351,11 +355,13 @@ def link_graph(self, linker, flat_graph):
         linked_graph = {
             'nodes': {},
             'macros': flat_graph.get('macros'),
+            'operations': flat_graph.get('operations'),
         }
 
-        for name, node in flat_graph.get('nodes').items():
-            self.link_node(linker, node, flat_graph)
-            linked_graph['nodes'][name] = node
+        for node_type in ['nodes', 'operations']:
```

Comment:
yeah, i think you're right -- these can in the `nodes` graph... let me investigate